be/src/olap/rowset/segment_v2/column_reader.cpp

```diff
@@ -536,17 +518,17 @@ Status DefaultValueColumnIterator::init(const ColumnIteratorOptions& opts) {
     return Status::OK();
 }
 
-Status DefaultValueColumnIterator::next_batch(size_t* n, ColumnBlock* dst) {
+Status DefaultValueColumnIterator::next_batch(size_t* n, ColumnBlockView* dst) {
+    if (dst->is_nullable()) {
+        dst->set_null_bits(*n, _is_default_value_null);
+    }
+
     if (_is_default_value_null) {
-        for (int i = 0; i < *n; ++i) {
-            dst->set_is_null(i, true);
-        }
+        dst->advance(*n);
     } else {
         for (int i = 0; i < *n; ++i) {
-            memcpy(dst->mutable_cell_ptr(i), _mem_value.data(), _value_size);
-            if (dst->is_nullable()) {
-                dst->set_is_null(i, false);
-            }
+            memcpy(dst->data(), _mem_value.data(), _value_size);
+            dst->advance(1);
```

Comment:
why 1?

Reply:
it's in the for loop, we memcpy and advance one value every time